test/integrations/testTypes.ts

```diff
@@ -44,3 +45,98 @@ export type MockHttpCallsData = {
   httpReq: Record<string, any>;
   httpRes: Partial<AxiosResponse>;
 };
+
+export const arrayOrSingleElement = (schema: ZodTypeAny) => z.union([z.array(schema), schema]);
+export const formElementSchema = arrayOrSingleElement(
+  z.union([z.string(), z.number(), z.boolean()]),
+);
+export const eventResponseSchema = z.union([
+  z.object({
+    error: z.string(),
+    statTags: z
+      .object({
+        destType: z.string().optional(),
+        errorCategory: z.string().optional(),
+        errorType: z.string().optional(),
+        feature: z.string().optional(),
```

Comment:
Can we equate this to be one of `FEATURES` in `src/v0/util/tags.js` that we already have ?

Reply:
this is done